packages/ui/src/components/atoms/Avatar/index.tsx

```diff
@@ -1,30 +1,56 @@
 import { FC, HTMLAttributes } from 'react'
-import styled from 'styled-components'
+import styled, { Interpolation, ThemedStyledProps } from 'styled-components'
 import { colors, device } from '../../../styles'
 import defaultAvatar from './default.svg'
 
 export type TAvatar = HTMLAttributes<HTMLDivElement> & {
   src?: string
   alt: string
+  avatarCss?: Interpolation<ThemedStyledProps<TAvatar, unknown>>
+  mediaQuery?: { [key: string]: string; }
+  forwardedRef?: React.RefObject<HTMLImageElement>
+  onClick?: () => void
 }
 
 const StyledAvatar = styled.img<TAvatar>`
-  width: 97px;
-  height: 97px;
-  border-radius: 50%;
+ ${props => props.avatarCss};
   object-fit: cover;
   appearance: auto;
   text-align: center;
-  background-color: ${colors.gray.gray5};
-  border: 1px solid ${colors.gray.gray4};
-
+  cursor: pointer;
   @media only ${device.Tablet} {
-    width: 118px;
-    height: 118px;
-    border: none;
+    ${(props) => props.mediaQuery}
   }
 `
+const defaultMediaQuery = {
+  width: '118px',
+  height: '118px',
+  border: 'none',
+}
+const defaultAvatarCss = {
+  width: '97px',
+  height: '97px',
+  borderRadius: '50%',
+  backgroundColor: colors.gray.gray5,
+  border: `1px solid ${colors.gray.gray4}`,
 
-export const Avatar: FC<TAvatar> = ({ src = defaultAvatar, alt }) => (
-  <StyledAvatar src={src} alt={alt} />
+}
+export const Avatar: FC<TAvatar> = ({
+  src = defaultAvatar,
+  alt,
+  avatarCss = {
+    defaultAvatarCss
+  },
+  forwardedRef,
+  mediaQuery = { ...defaultMediaQuery },
+  onClick
+}) => (
+  <StyledAvatar
+    src={src}
+    alt={alt}
+    avatarCss={avatarCss}
+    ref={forwardedRef}
+    mediaQuery={mediaQuery}
+    onClick={onClick}
```

Comment:
What's mediaQuery prop for? It seems to repeat/contradict avatarCss ¿?

Reply:
mediaquery are the variable that will be used when passing diferents @media depending in where is going to be used, for navbar need some media style and card profiles a diferent one, I took them from the original file, now that userbtn will be removed.